isthmus/src/test/java/io/substrait/isthmus/SimplePlansTest.java

```diff
@@ -1,59 +1,53 @@
 package io.substrait.isthmus;
 
-import com.google.common.collect.Lists;
-import com.google.protobuf.InvalidProtocolBufferException;
-import com.google.protobuf.util.JsonFormat;
-import io.substrait.proto.Plan;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.List;
 import org.apache.calcite.sql.parser.SqlParseException;
 import org.junit.jupiter.api.Test;
 
 public class SimplePlansTest extends PlanTestBase {
 
   @Test
   public void aggFilter() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    s.execute("select sum(L_ORDERKEY) filter(WHERE L_ORDERKEY > 10) from lineitem ", creates);
+    assertProtoPlanRoundrip("select sum(L_ORDERKEY) filter(WHERE L_ORDERKEY > 10) from lineitem ");
   }
 
   @Test
   public void cd() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    // creates.forEach(System.out::println);
-    s.execute(
-        "select l_partkey, sum(distinct L_ORDERKEY) from lineitem group by l_partkey ", creates);
+    assertProtoPlanRoundrip(
+        "select l_partkey, sum(distinct L_ORDERKEY) from lineitem group by l_partkey ");
   }
 
   @Test
   public void filter() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    // creates.forEach(System.out::println);
-    print(s.execute("select * from lineitem WHERE L_ORDERKEY > 10", creates));
+    assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY > 10");
   }
 
   @Test
-  public void joinWithMultiDDLInOneString() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    List<String> ddlStmts = Lists.newArrayList(asString("tpch/schema.sql"));
-    print(
-        s.execute(
-            "select * from lineitem l, orders o WHERE o.o_orderkey = l.l_orderkey  and L_ORDERKEY > 10",
-            ddlStmts));
+  public void trailingSemicolon() throws IOException, SqlParseException {
+    assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY > 10;");
   }
 
-  private void print(Plan plan) {
+  @Test
+  public void multiStatement() throws IOException, SqlParseException {
     try {
```

Comment:
I believe there is a new lambda canonical way to do exception checking using junit.

Reply:
This tests both that it fails and succeeds without vs with flag.

Reply:
But I can have two tests if you think that's better

Reply:
I'm not arguing for two separate tests. Just saying that something like this would be more idiomatic:

```
assertThrows(UnsupportedOperationException.class, () -> {
  assertProtoPlanRoundrip("select l_orderkey from lineitem; select l_partkey from lineitem WHERE L_ORDERKEY > 20;");
}, "SQL must contain only a single statement");
```

Reply:
Updated